app/jobs/clock/refresh_wallets_ongoing_balance_job.rb

```diff
@@ -11,7 +11,7 @@ class RefreshWalletsOngoingBalanceJob < ApplicationJob
     def perform
       return unless License.premium?
 
-      Wallet.active.find_each do |wallet|
+      Wallet.active.ready_to_be_refreshed.find_each do |wallet|
         Wallets::RefreshOngoingBalanceJob.perform_later(wallet)
```

Comment:
could be useful to check if the boolean is still set in the actual job as well? Prevents sidekiq from processing wallets (that no longer need to be refreshed, due to already being processed etc.)

Reply:
good spot - thanks 👍